physics/module_SGSCloud_RadPre.F90

```diff
@@ -241,55 +274,92 @@ subroutine sgscloud_radpre_run(    &
         endif ! end MYNN or OTHER choice for background clouds fractions
 
         ! At this point, we have cloud properties for all non-deep convective clouds.
-        ! So now we add the convective clouds,
+        ! So now we add the convective clouds:
 
         if (imfdeepcnv == imfdeepcnv_gf) then
           do k = 1, levs
             do i = 1, im
               !if ( qci_conv(i,k) > 0. .AND. (qi(i,k) < 1E-7 .AND. qc(i,k) < 1E-7 ) ) then
               if ( qci_conv(i,k) > 0. ) then
-                !Partition the convective clouds into water & ice according to a linear
-                qc(i,k) = qc(i,k)+qci_conv(i,k)*(min(1., max(0., (T3D(i,k)-244.)/25.)))
-                qi(i,k) = qi(i,k)+qci_conv(i,k)*(1. - min(1., max(0., (T3D(i,k)-244.)/25.)))
+                Tk = T3D(i,k)
+                Tc = Tk - 273.15
 
-                Tc = T3D(i,k) - 273.15
+                !Partition the convective clouds into water & frozen species
+                liqfrac = min(1., max(0., (Tk-244.)/29.))
```

Comment:
I noticed that _min(1., max(0., (T3D(i,k)-244.)/**25**.))_ switched to _min(1., max(0., (Tk-244.)/**29**.))._ Is the switch from 25 to 29 in the dominator done purposely, or is it a typo?

Reply:
It was done on purpose, but I have plenty of doubt as to which is optimal. In reality, the partitioning of water vs frozen condensate is more than a function of temperature, so both values are wrong. 

In this case, since most condensate between 0 and -4 C is in water phase, maybe we should change it back to 25. If you prefer it changed to 25, I 'll do it.

This tweak won't require new regression tests.